src/plugins/initiatives/initiativeFile.js

```diff
@@ -19,6 +20,28 @@ export const INITIATIVE_FILE_SUBTYPE = "INITIATIVE_FILE";
  */
 export type InitiativeFile = InitiativeFileV020;
 
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeFileV100 = {|
+  id: string, // json file name right now. probably not needed if this is stored in side a json db file
+  // Compat info is usually stored within a header object, but in the db file parsing is more
+  // straightforward if it's folded inside the primary data object
+  type: string,
+  version: string,
+  title: string,
+  timestampIso: TimestampISO,
```

Comment:
Let's use `TimestampMs` for consistency with all our other serialized timestamps, we were only using `TimestampISO` for human read-ability on the JSON files.